backtrace-library/src/main/cpp/backends/crashpad-backend.cpp

```diff
@@ -0,0 +1,206 @@
+#include "crashpad-backend.h"
+#include "backtrace-native.h"
+#include <jni.h>
+#include <libgen.h>
+
+extern std::string thread_id;
+extern std::atomic_bool initialized;
+extern std::mutex attribute_synchronization;
+
+bool InitializeCrashpad(jstring url,
+                        jstring database_path,
+                        jstring handler_path,
+                        jobjectArray attributeKeys,
+                        jobjectArray attributeValues,
+                        jobjectArray attachmentPaths,
+                        jboolean enableClientSideUnwinding,
+                        jint unwindingMode) {
+    using namespace crashpad;
```

Comment:
We barely use the names from inside this namespace, maybe just use fully-qualified names instead?

Reply:
Agreed, will do!